modules/exploits/unix/webapp/wp_phpmailer_host_header.rb

```diff
@@ -0,0 +1,176 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HTTP::Wordpress
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'                => 'WordPress PHPMailer Host Header Command Injection',
+      'Description'         => %q{
+        This module exploits a command injection vulnerability in WordPress
+        version 4.6 with Exim as an MTA via a spoofed Host header to PHPMailer.
+
+        A valid WordPress username is required to exploit the vulnerability.
+        Additionally, due to the altered Host header, exploitation is limited to
+        the default virtual host, assuming the header isn't mangled in transit.
```

Comment:
Updated the PR description. Kinda surprised I didn't notice it! I guess I should have read what I pasted. :D